easybuild/easyconfigs/a/anndata/anndata-0.11.1-foss-2023b.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'anndata'
+version = '0.11.1'
+
+homepage = 'https://github.com/scverse/anndata'
+description = """anndata is a Python package for handling annotated data matrices in memory and on disk,
+ positioned between pandas and xarray"""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+builddependencies = [
+    ('hatchling', '1.18.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('SciPy-bundle', '2023.11'),
+    ('h5py', '3.11.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('array_api_compat', '1.9.1', {
+        'checksums': ['17bab828c93c79a5bb8b867145b71fcb889686607c5672b060aef437e0359ea8'],
+    }),
+    ('natsort', '8.4.0', {
+        'checksums': ['45312c4a0e5507593da193dedd04abb1469253b601ecaf63445ad80f0a1ea581'],
+    }),
+    (name, version, {
+        'checksums': ['36bff9a85276fc5f1b9fd01f15aff9aa49408129985f42e0fca4e2c5b7fa909f'],
+    }),
+]
+
```

Comment:
@pavelToman could you restore the `sanity_check_paths`, please.

Reply:
I can, but since the `lib/python%(pyshortver)s/site-packages` is checked anyway and `bin/nasort` is checked by working `natsort --help` I thought it is not necessary.